src/sagas/index.js

```diff
@@ -689,6 +690,9 @@ export function* stopProgress ({ vmId, poolId, name, result }) {
 function* shutdownVm (action) {
   yield startProgress({ vmId: action.payload.vmId, name: 'shutdown' })
   const result = yield callExternalAction('shutdown', Api.shutdown, action)
+  if (result.status === 'complete') {
+    yield put(addUserMessage({ message: 'VM Shutdown Request Received', type: 'success' }))
```

Comment:
The text needed to be externalized (moved to `messages.js`).

I suggest naming the message key `toastVmShutdown`.